freqtrade/rpc/rpc.py

```diff
@@ -136,7 +136,11 @@ def _rpc_show_config(config, botstate: Union[State, str],
             'ask_strategy': config.get('ask_strategy', {}),
             'bid_strategy': config.get('bid_strategy', {}),
             'state': str(botstate),
-            'runmode': config['runmode'].value
+            'runmode': config['runmode'].value,
+            'position_adjustment_enable': config.get('position_adjustment_enable', False),
+            'max_buy_position_adjustment': (config['max_buy_position_adjustment']
```

Comment:
unfortunately, the way the defaults are set, this causes a crash in webserver mode (start freqtrade with `freqtrade webserver` with a config that does not contain this key and login to the UI).

Not all freqtrade modes load a strategy right away - so to properly set the default, we'll have to have it as part of the jsonschema (including a default value) in `constatnts.py`.